### PR TITLE
Simplifies dependency graph by only focusing on callCount

### PR DESCRIPTION
**Pull request by codefromthecrypt**
One of the challenges of resurrecting the Dependency Graph feature has
been the incidental complexity of Algebird/Moments. The moments api
obfuscated otherise simple relationships such as how many calls are made
between links.

This removes Algebird from the data model and flattens code and schema
around it. This will significantly ease our process to apply dependency
graph to all span stores, while not excluding more advanced analysis
from occurring in the future.

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
screen shot 2015-09-23 at 1 04 59 pm

Still works! i.e. the darkness of the line corresponds to call count.

File: gradle.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #Sat, 15 Aug 2015 15:15:18 +0000
-version=1.9.1-SNAPSHOT
+version=1.10.0-SNAPSHOT
```
Comment (codefromthecrypt) [inline gradle.properties:2]: bumped as this is a schema affecting change

**Comment by spencergibb** [comment]
Nice, that removes a ton of complexity. :+1:

**Comment by kristofa** [comment]
Besides the build that is not happy (not sure if related to change or flaky test) :+1:

**Comment by yurishkuro** [comment]
LGTM

**Pull request merged**